dashboards.kubernetes/pulsar_detector.json

```diff
@@ -0,0 +1,317 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 0,
+  "id": 1,
+  "links": [],
+  "panels": [
+    {
+      "cacheTimeout": null,
+      "datasource": null,
```

Comment:
Can you add "{{ PULSAR_CLUSTER }}" to the data source? So the data source can be replaced with the right data source when running on Kubernetes.

Reply:
Example: https://github.com/streamnative/apache-pulsar-grafana-dashboard/blob/master/dashboards.kubernetes/bookkeeper.json#L83